### PR TITLE
Visual Basic support for binary literals

### LINKED ISSUE #1963: Document VB support for binary, hex literals, and digit separator

### PR DESCRIPTION
**Pull request by rpetrusha**
# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

This documents support for binary literals added to VB 2017.

Fixes #1963

This is a work in progress -- I'll tag it with the WIP label, and remove it when the documentation revision is complete.

## Suggested Reviewers

@AnthonyDGreen @BillWagner

### CODE REVIEW

**Comment by rpetrusha** [comment]
@BillWagner @AnthonyDGreen This is ready for review, if you can take a look.

**Review by AnthonyDGreen — APPROVED** [review]
I just looked at the updates for Byte and they look good. I would suggest updating the code examples to use & for string concatenation instead of + or even eliminating the `vbCrLf +` bits entirely. They're a bit distracting from the topic, I think.

File: docs/visual-basic/language-reference/data-types/sbyte-data-type.md [hunk 1]
```diff
@@ -43,25 +43,51 @@ translation.priority.ht:
...
 ---
-# SByte Data Type (Visual Basic)
+# SByte data type (Visual Basic)
+
 Holds signed 8-bit (1-byte) integers that range in value from -128 through 127.  
   
 ## Remarks  
- Use the `SByte` data type to contain integer values that do not require the full data width of `Integer` or even the half data width of `Short`. In some cases the common language runtime might be able to pack your `SByte` variables closely together and save memory consumption.  
+
+Use the `SByte` data type to contain integer values that do not require the full data width of `Integer` or even the half data width of `Short`. In some cases the common language runtime might be able to pack your `SByte` variables closely together and save memory consumption.
```
Comment (mairaw) [inline docs/visual-basic/language-reference/data-types/sbyte-data-type.md]: nit: add comma after in some cases

**Comment by AnthonyDGreen** [comment]
Thanks, @rpetrusha!

**Comment by gafter** [comment]
@AnthonyDGreen This PR confuses me. Where is the spec for VB 14? Where is the spec for VB 15? What version is intended to be documented by the main branch here?

**Pull request merged**